### PR TITLE
update template

### PR DESCRIPTION
**Pull request by olga-f**
# Don't forget to add Jira issue related code to the title 😇

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix

## Description

- Added Base Web design system configuration
- Added Google Tag Manager configuration
- Added tests
- Added GitHub workflow configuration
- Updated Readme

### CODE REVIEW

**Review by romerox3 — APPROVED** [review]
It's fine for me! 👏  Only the line breaks that I comment you

File: src/components/main.tsx [hunk 1]
```diff
@@ -1,12 +1,33 @@
+import { Card, StyledBody } from 'baseui/card';
+import { Display2 } from 'baseui/typography';
+import { H3 } from 'baseui/typography';
+import { useStyletron } from 'baseui';
 const Main = (): JSX.Element => {
```
[the remark this answers is not in the document]
Comment (olga-f) [reply src/components/main.tsx:6]: Added. Thanks, Antonio!

File: src/pages/_document.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  Html,
+  Main,
+  NextScript
+  } from 'next/document';
+import { GTM_ID } from '../lib/gtm';
+import { Provider as StyletronProvider } from 'styletron-react';
+import { styletron } from '../lib/styletron';
+import type { DocumentContext } from "next/document";
+import type { Key } from "react";
+class MyDocument extends Document {
```
[the remark this answers is not in the document]
Comment (olga-f) [reply src/pages/_document.tsx:13]: 👌

**Review by emege93 — APPROVED** [review]
GJ! 🚀

File: src/helpers/themeHelper.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { createTheme } from 'baseui';
+import { overrides, primitives } from '../utils';
+
+
+export const theme = createTheme(primitives, overrides);
```
Comment (anthares101) [inline src/helpers/themeHelper.ts:5]: Just a silly thing,in this file there is an extra line break :eyes:
```suggestion
import { createTheme } from 'baseui';
import { overrides, primitives } from '../utils';

export const theme = createTheme(primitives, overrides);
```

**Review by anthares101 — CHANGES_REQUESTED** [review]

**Review by anthares101 — APPROVED** [review]

File: src/helpers/themeHelper.ts (hunk 3 above)
Comment (olga-f) [reply src/helpers/themeHelper.ts]: Thanks! Done.

**Pull request merged**